src/crepe/api/Script.h

```diff
@@ -46,9 +47,17 @@ class Script {
 	/**
 	 * \brief Script update function (empty by default)
 	 *
+	 * \param delta_time Time since last fixed update
+	 *
 	 * This function is called during the ScriptSystem::update() routine if the \c BehaviorScript
 	 * component holding this script instance is active.
 	 */
+	virtual void update(duration_t delta_time) { return this->update(); }
```

Comment:
Als je `void update(duration_t delta_time)` override, dan wordt `void update()` niet meer aangeroepen. Dit is niet wenselijk/wat je verwacht.
Graag alleen `void update(duration_t delta_time)` ondersteunen óf zowel `void update(duration_t delta_time)` én `void update()` aanroepen.